samples/tree.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Write your Calder code here
+ */
+
+const light1 = {
+    lightPosition: [10, 10, 10],
+    lightColor: [0.3, 0.3, 0.3],
+    lightIntensity: 256
+};
+const light2 = {
```

Comment:
Would having a `Light` class be more explicit about what we're creating rather than just a generic `JSON` object which implicitly requires these three fields?